### PR TITLE
Create Action not working as expected

### PR DESCRIPTION
**Pull request by praveenyr**
**Description:**  The _Create_ CRUD action flashes an error when I try to create a wiki through the app.The rspec tests work fine for the same though.

Test Coverage : 89.38%

### CODE REVIEW

File: app/controllers/wikis_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def show
+  end
+
+  def new
+    @wiki = Wiki.new
+  end
+  
+  def create
+    @wiki = Wiki.new(wiki_params)
+    @wiki.user_id = params[:wiki][:user]
```
Comment (brittanyjspringer) [inline app/controllers/wikis_controller.rb]: Change this to: 

@wiki.user_id = params[:wiki][:user] ||= current_user.id

That way, if the user ID is not sent, we set the user to the current user. This works and passes the tests.

File: spec/helpers/wikis_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/wikis_helper_spec.rb]: Please delete any helper or view tests to keep the test runs clean :)

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by brittanyjspringer** [comment]
@praveenyr back to you! See comments above.

File: app/controllers/wikis_controller.rb (hunk 1 above)
Comment (praveenyr) [reply app/controllers/wikis_controller.rb]: Cool,that worked.Thanks Brittany!

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**